File: Fibonacci Examples/fib.java
public class fib {
    public static int fib(int n) {
        if(n==0){
            return 0;
        }
        if(n==1){
            return 1;
        }
        return fib(n-1) + fib(n-2);
        
        
    }
    public static void main(String[] args) {
        for(int i=0;i<10;i++){
            System.out.println(fib(i));
            float ratio = fib(i)/fib(i+1);
            System.out.println(ratio);
        }
        
    }
}

File: Fibonacci Examples/fib.py
def fib(num1):
    if num1==0:
        return 0
    elif num1==1:
        return 1
    else:
        return fib(num1-1) + fib(num1-2)

for i in range(10):
    print(fib(i))

values =[]
for i in range(10):
    values.append(i)
print (values)